app/index.js

```diff
@@ -1,4 +1,8 @@
 // simple hello world JS
 console.log("Hello World!!!");
 
+var test = 1;
+
+sample = "sample"
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "sample" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=javascript%3AS2703)